docs/aws.md

```diff
@@ -44,16 +44,44 @@ brew install kubernetes-cli
 
 ## Setup your environment
 
-### Setting up a kops IAM user
+## Create an IAM user for kops
```

Comment:
yeah, i had that same issue with the way it reads, wasn't sure of the best place to move this stuff to.

I'll make that change and have it pushed momentarily.